demisto_sdk/commands/common/errors.py

```diff
@@ -238,6 +239,16 @@ def invalid_id_set():
                "From content repository root run the following: `rm -rf Tests/id_set.json`\n" \
                "Then re-run the `validate` command."
 
+    @staticmethod
+    @error_code_decorator
+    def no_minimal_fromversion_in_file(fromversion, oldest_supported_version):
+        if fromversion == 'fromversion':
+            return f"{fromversion} field is invalid.\nAdd `{fromversion}: " \
+                   f"{oldest_supported_version}` to the file."
+        else:
+            return f'{fromversion} field is invalid.\nAdd `"{fromversion}": "{oldest_supported_version}"` ' \
+                   f'to the file.'
```

Comment:
what is the difference between the cases?

Reply:
the use of `"` - I want to be written as a simple copy-paste solution